### ISSUE TITLE
Debuggers aren´t working anymore

### ISSUE THREAD

**Issue by SalokinGreen** [microsoft/vscode#110639]
Issue Type: <b>Bug</b>

Most debuggers aren´t listed and they which are listed don´t work

VS Code version: Code 1.51.1 (e5a624b788d92b8d34d1392e4c4d9789406efe8f, 2020-11-10T23:34:32.027Z)
OS version: Windows_NT x64 10.0.18363

<details>
<summary>System Info</summary>

|Item|Value|
|---|---|
|CPUs|Intel(R) Core(TM) i7-7700 CPU @ 3.60GHz (8 x 3600)|
|GPU Status|2d_canvas: enabled<br>flash_3d: enabled<br>flash_stage3d: enabled<br>flash_stage3d_baseline: enabled<br>gpu_compositing: enabled<br>multiple_raster_threads: enabled_on<br>oop_rasterization: disabled_off<br>opengl: enabled_on<br>protected_video_decode: unavailable_off<br>rasterization: enabled<br>skia_renderer: disabled_off_ok<br>video_decode: enabled<br>vulkan: disabled_off<br>webgl: enabled<br>webgl2: enabled|
|Load (avg)|undefined|
|Memory (System)|31.96GB (15.44GB free)|
|Process Argv|--crash-reporter-id a1d23ad2-5370-4a74-8fe7-73356a80fc8b|
|Screen Reader|no|
|VM|0%|
</details><details><summary>Extensions (5)</summary>

Extension|Author (truncated)|Version
---|---|---
vscode-express|Com|0.0.5
csharp|ms-|1.23.6
python|ms-|2020.11.358366026
jupyter|ms-|2020.11.358541065
cpptools|ms-|1.1.1

</details>

**Comment by weinand** [comment]
@SalokinGreen 
> Most debuggers aren´t listed 

What do you mean? Debuggers listed where?

> which are listed don´t work

Please be specific and provide reproducible steps.

**Comment by SalokinGreen** [comment]
@weinand I´m sorry for that. When I pressed F5 (started debugging), there was a list of all available debuggers and I could choose between them. Now there is only google chrome and Node.js preview. Both aren´t working.

For example, if I write a program to use with node.js and I use the "Node.js preview" debugger, nothing happens. 
I got the same problem with Python and others. It isn´t the fault of the code because it works with other Editors than VS Code.
